lib/interface/cli/completion/dynamic/Pipeline.completion.js

```diff
@@ -0,0 +1,10 @@
+const getPipelines = async (word, argv) => {
+    const { pipeline } = require('../../../../logic').api;
+
+    const pips = await pipeline.getAll({ limit: 25, offset: 0 });
```

Comment:
please put dynamic auto cmpletion files next to the matching resource folder.
for example, inside pipeline folder in some way